src/components/Shared/Table.css

```diff
@@ -6,4 +6,8 @@ table.responsiveTable {
 table.responsiveTable td {
     word-wrap: break-word !important;
 }
-    
\ No newline at end of file
+
+table.responsiveTable thead {
```

Comment:
you can change it to `table.responsiveTable > thead` since thead will be the child